src/PropertyDetail.js

```diff
@@ -1,33 +1,40 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import {useParams} from "react-router-dom";
-import lowConditionParcelsData from './data/lowPoorWornOutConditionParcels.json'
-import outOfCityParcelsData from "./data/ownerInNebraskaOutOfOmahaParcels.json";
-import outOfNebraskaParcelsData from "./data/ownerOutOfNebraskaParcels.json";
 import Card from '@material-ui/core/Card';
 import CardContent from '@material-ui/core/CardContent';
 import Typography from '@material-ui/core/Typography';
 import Link from '@material-ui/core/Link';
 import Box from '@material-ui/core/Box';
 import Breadcrumbs from '@material-ui/core/Breadcrumbs';
+import ReactGA from "react-ga";
+import lowConditionParcelsData from './data/lowPoorWornOutConditionParcels.json'
+import outOfCityParcelsData from "./data/ownerInNebraskaOutOfOmahaParcels.json";
+import outOfNebraskaParcelsData from "./data/ownerOutOfNebraskaParcels.json";
 
 export default function PropertyDetail() {
 
-  const {source, pin} = useParams();
-  let selectedProperty = {};
-  let propertyCount = 0;
-  if (source === "out-of-state") {
-    selectedProperty = outOfNebraskaParcelsData.find(property => property.PIN === pin);
-    propertyCount = outOfNebraskaParcelsData.filter(property => property.OWNER_NAME === selectedProperty.OWNER_NAME).length;
-  } else if (source === "out-of-omaha") {
-    selectedProperty = outOfCityParcelsData.find(property => property.PIN === pin);
-    propertyCount = outOfCityParcelsData.filter(property => property.OWNER_NAME === selectedProperty.OWNER_NAME).length;
-  } else if (source === "low-condition") {
-    selectedProperty = lowConditionParcelsData.find(property => property.PIN === pin);
-    propertyCount = lowConditionParcelsData.filter(property => property.OWNER_NAME === selectedProperty.OWNER_NAME).length;
-  }
+  const [selectedProperty, setSelectedProperty] = useState({QUALITY: "", CONDITION: ""});
```

Comment:
This was a bit unexpected.  Later the `setSelectedProperty` function changes the shape of this state.
There are a lot of values on the `property` object that are used later. Does setting these two mean something specific?

Reply:
I needed defaults because the {{.trim()}} that happens in the render.